src/components/scenes/PinLoginScene.tsx

```diff
@@ -91,17 +101,33 @@ export function PinLoginScene(props: Props) {
     [dropdownItems, loginId]
   )
 
-  const displayUsername = React.useMemo(
-    () =>
-      userInfo?.username ??
-      sprintf(lstrings.guest_account_id_1s, loginId.slice(loginId.length - 3)),
-    [loginId, userInfo]
-  )
-
   // ---------------------------------------------------------------------
   // Effects
   // ---------------------------------------------------------------------
 
+  const handleTouchLogin = async (userInfo: EdgeUserInfo): Promise<void> => {
```

Comment:
Why was this moved to the "Effects" section? It's a handler, so it should stay in the "Handlers" section. I don't see any dependencies that have changed, so the location shouldn't matter.

Reply:
I was getting a error about using before declaration